python/lsst/meas/base/references.py

```diff
@@ -179,21 +206,20 @@ def validate(self):
 
 
 class CoaddSrcReferencesTask(BaseReferencesTask):
-    """!
-    A references task implementation that loads the coadd_datasetSuffix dataset directly from
+    """A references task implementation that loads the coadd_datasetSuffix dataset directly from
     disk using the butler.
     """
 
     ConfigClass = CoaddSrcReferencesConfig
     datasetSuffix = "src"  # Suffix to add to "Coadd_" for dataset name
 
     def __init__(self, butler=None, schema=None, **kwargs):
-        """! Initialize the task.
-        Additional keyword arguments (forwarded to BaseReferencesTask.__init__):
-         - schema: the schema of the detection catalogs used as input to this one
-         - butler: a butler used to read the input schema from disk, if schema is None
-        The task will set its own self.schema attribute to the schema of the output merged catalog.
-        """
+        # """! Initialize the task.
```

Comment:
Like the others, delete the `__init__`'s docstring, but move the discussion of the keyword arguments to the Parameters section of the class docstring.